LICENSE.md

```diff
@@ -0,0 +1,22 @@
+Copyright (c) 2018-2020 Michalis Zampetakis
+Copyright (c) 2015-2020 Thanasis Zografos
+Copyright (c) 2015-2020 Antonios Chariton
+Copyright (c) 2015-2020 Nikos Kampitakis
+Copyright (c) 2019-2020 Giannis Koutsaftakis
+Copyright (c) 2016-2020 Yorgos Saslis
```

Comment:
if the start dates are different, would end dates be different as well? 

(been a while since i committed anything here...)

Reply:
Yes, in theory.

I think that I would just remove the "personalized" copyright notice and just add the license. Contributors can be seen in gh and through git logs...
WDYT?

Reply:
Thanks, done.